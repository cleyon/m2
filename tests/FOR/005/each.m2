@@ Keys might be output in arbitrary order but in practice
@@ they seem stable enough for testing
@array A
@define A[foo] Foo
@define A[bar] Bar
@define A[baz] Baz
Items may print in different order - that's okay
@foreach I A
I=@I@; @A[@{I}]@
@next I
