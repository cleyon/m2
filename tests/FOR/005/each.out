Items may print in different order - that's okay
I=baz; Baz
I=foo; Foo
I=bar; Bar
